requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy
+tensorflow
+keras
+https://github.com/csxeba/grund
```

Comment:
Syntax here is almost the same as in the readme: `git+https://github.com/csxeba/grund.git#egg=grund`

https://pip.readthedocs.io/en/1.1/requirements.html